### PR TITLE
deps: extension-port-stream@^2.0.0->^3.0.0

### PR DESCRIPTION
**Pull request by legobeat**
- Update `extension-port-stream` from `^2.0.0` to [`^3.0.0`](https://github.com/MetaMask/extension-port-stream/releases/tag/v3.0.0)
  - [Changelog](https://github.com/MetaMask/extension-port-stream/blob/v3.0.0/CHANGELOG.md)
  - Corresponding update in `metamask-extension`: https://github.com/MetaMask/metamask-extension/pull/21931

#### Blocked by
- [x] #36

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -390,11 +1154,178 @@
...
+        "glob-to-regexp": "^0.4.1",
+        "graceful-fs": "^4.1.2"
+      }
+    },
+    "webextension-polyfill": {
+      "version": "0.11.0",
+      "resolved": "https://registry.npmjs.org/webextension-polyfill/-/webextension-polyfill-0.11.0.tgz",
+      "integrity": "sha512-YUBSKQA0iCx2YtM75VFgvvcx1hLKaGGiph6a6UaUdSgk32VT9SzrcDAKBjeGHXoAZTnNBqS5skA4VfoKMXhEBA==",
+      "requires": {
+        "webpack": "^5.91.0",
```
Comment (Gudahtt) [inline package-lock.json:1239]: Welp. This isn't ideal. Seems like a mistake: https://github.com/mozilla/webextension-polyfill/issues/604

Comment (Gudahtt) [reply package-lock.json:1239]: Maybe we can use v0.10.0 until they fix that issue

Comment (legobeat) [reply package-lock.json]: Good catch! Lockfile update made separate in #36.

Comment (Gudahtt) [reply package-lock.json]: This is now fixed upstream, in v0.12.0

Comment (legobeat) [reply package-lock.json]: Nice job!

**Comment by legobeat** [comment]
@SocketSecurity ignore npm/extension-port-stream@3.0.0

new author ok

<details>
selfaward
</details>

**Comment by legobeat** [comment]
Merging in lack of reviewer response.

**Pull request merged**